scripts/release/rpm/sync_repo.py

```diff
@@ -102,9 +102,10 @@ def main(argv=sys.argv[1:]):
             print('Syncing %d packages from %s to %s...%s' % (
                 len(packages_to_sync), dist_source, dist_dest,
                 ' (dry run)' if args.dry_run else ''))
+            package_cache = dict(packages_to_sync)
```

Comment:
Is the purpose of this line is to duplicate the packages_to_sync list to use it as the package_cache?

Is that worth a comment to ensure that this line isn't later removed in a cleanup attempt or will the commit history provide significant justification for an attentive maintainer?

Reply:
Also in an offline discussion @cottsay and I were talking about setting up a PackageCache object if the cache is meant to be a living object that is updated by multiple different functions rather than a convenient primitive to deliver data in a unidirectional way.

Reply:
> Is the purpose of this line is to duplicate the packages_to_sync list to use it as the package_cache?

It is. I think the line is explicit enough that commit history should suffice.

> setting up a PackageCache object

This code is likely to change heavily when we update to a newer Pulp release and take advantage of some of the new features. I'll defer this refactor until then (if the concept even carries forward).